common/libs/transaction.ts

```diff
@@ -75,11 +75,12 @@ export async function generateCompleteTransactionFromRawTransaction(
   node: INode,
   tx: ExtendedRawTransaction,
   wallet: IWallet,
-  token: Token | null | undefined
+  token: Token | null | undefined,
+  skipEthAddressValidation: boolean
 ): Promise<CompleteTransaction> {
   const { to, data, gasLimit, gasPrice, from, chainId, nonce } = tx;
   // Reject bad addresses
-  if (!isValidETHAddress(to)) {
+  if (!isValidETHAddress(to) && !skipEthAddressValidation) {
```

Comment:
Note: Seems dangerous, but I don't have a better alternative.

Reply:
Ideally this would be addressed in a `transactions.ts` update where we can decouple user validation vs transaction validation. As the above would be a (unless they wanted to burn their ether) invalid user tx, but a valid transaction still.